### DISCUSSION TITLE
Table Name Issue for oracle Cloud database

### DISCUSSION THREAD

**Discussion by amitdumka** [oracle/python-oracledb#293] [Q&A] [answered]
I am an asp dot net project which is using ef core to manage database.
so it create tables in Oracle cloud database hosted at oracle cloud. 
table names are like Employees, Roaster, Users, Contacts, Persons. etc. 
Now i want to write python script to fetch from data from these tables , its return error that "ORA-00942: table or view does not exist". 
If i create table directly from cloud sql console or from python, tables are created like these name "EMPLOYEES, ROASTER,USERS, CONTACTS", and i am to fetch data from these name. 

So I want to know it is possible to access and fetch data from tables which is this format "%<A-Z>%<a-z>..." i.e. Employees. 
i tried many ways but nothing works nor i am able to find any solutions any where. 
Kindly suggest or guide.
option to fetch data from python, and  c# web api with efcore.

**Comment by anthony-tuininga** [comment, marked as answer]
I'm not sure what you mean, precisely. If the table name is ``EMPLOYEES`` (as shown in the database catalog) then you can issue this query without difficulties:

```python
cursor.execute('select * from Employees')
```

but if the table name is ``Employees`` (as shown in the database catalog) then you must do this intead:

```python
cursor.execute('select * from "Employees"')
```

If I missed the question you were trying to ask, please clarify!

**Reply by amitdumka** [reply]
ok this solve my problem , Thanks for answer
